<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>573</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>5</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>수입예산 입력</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>예산 년도:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="budget_year_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>48</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>66</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="last_order_tableWidget">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>170</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>헌금명칭</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="last_jeol_tableWidget">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>326</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>헌금명칭</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>146</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>전년도</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="this_order_tableWidget">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>170</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>헌금명칭</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전년대비</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>146</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>당해년도</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_year_order_amount">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>148</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="this_year_order_amount">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>148</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="this_jeol_tableWidget">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>324</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>헌금명칭</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전년대비</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>297</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>당해년도</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>전년도</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_year_jeolgi_amount">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>301</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="this_year_jeolgi_amount">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>301</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>수입예산 합계 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_year_hun_hap">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>520</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="this_year_hun_hap">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="last_this_ratio_widget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>145</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>예배헌금</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>299</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>절기헌금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="budgetl_save_Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장하기(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="budget_close_Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>45</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="QPushButton" name="budget_list_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>예산항목 가져오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reculculate_Button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>재계산</string>
   </property>
  </widget>
  <widget class="QLabel" name="order_ratio_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>305</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="jeol_ratio_label">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>305</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>82</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>전년 예산 보기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>budget_year_widget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>budget_list_Button</tabstop>
  <tabstop>this_order_tableWidget</tabstop>
  <tabstop>this_jeol_tableWidget</tabstop>
  <tabstop>this_year_order_amount</tabstop>
  <tabstop>this_year_jeolgi_amount</tabstop>
  <tabstop>last_order_tableWidget</tabstop>
  <tabstop>last_jeol_tableWidget</tabstop>
  <tabstop>last_year_order_amount</tabstop>
  <tabstop>last_year_jeolgi_amount</tabstop>
  <tabstop>reculculate_Button</tabstop>
  <tabstop>budgetl_save_Button</tabstop>
  <tabstop>budget_close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>budgetl_save_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>budget_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>budget_close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>budget_end_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>budget_list_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>budget_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reculculate_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>re_calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>serch_last_budget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>budget_save()</slot>
  <slot>budget_end_close()</slot>
  <slot>budget_list()</slot>
  <slot>re_calculate()</slot>
  <slot>serch_last_budget()</slot>
 </slots>
</ui>
